pkg/apprclient/adapter.go

```diff
@@ -0,0 +1,77 @@
+package apprclient
+
+import (
+	"bytes"
+
+	appr "github.com/operator-framework/go-appr/appregistry"
+	appr_blobs "github.com/operator-framework/go-appr/appregistry/blobs"
```

Comment:
nit: [underscores in package names?](https://golang.org/doc/effective_go.html#package-names)